docs/mnist_example/README.md

```diff
@@ -106,6 +106,53 @@ onnx-mlir -O3 --EmitJNI mnist.onnx
 
 A `mnist.jar` should appear, which corresponds to the compiled model object file along with Java API classes.
 
+## Multi-threading
+
+onnx-mlir provides a multi-thread safe parallel compilation mode. Whether each thread is given a name or not by the user, onnx-mlir is multi-threaded safe. If you would like to give a name to a thread, use the `-customEnvFlags` keyword and an example can be found as follows.
```

Comment:
> If you would like to give a name to a thread

I don't understand this. I though that argument was to replace the default `ONNX_MLIR_FLAGS` env var? In any case, because of how you constructed the `omCompileFromFileViaCommand` command, I am not sure we need to override the `ONNX_MLIR_FLAGS` env var anymore.

We can talk if easier to communicate on this.

Reply:
My pleasure to work on this. Thanks for your nice support and understanding since I know I take a long time to get familiar with it and finish it. I think I have already fixed most of the comments.

For this one, I was thinking that many examples under [test](https://github.com/onnx/onnx-mlir/tree/35c1a375f7d57645d86a7d7bcab244213416f0a3/test) folder pass a custom name which repalce the original `OnnxMlirEnvOptionName` in `onnx-mlir.cpp` while calling the `llvm::cl::ParseCommandLineOptions` function. So I want to keep such an interface to replace the default `ONNX_MLIR_FLAGS` env var for future use. That's why I didn't directly remove the `-customEnvFlags`.

Also I am happy to discuss via Slack but I think I have trouble accessing the Slack channel via the link given in the onnx-mlir README and I am not able to find you. Could you please add me there or somewhere else? My email account of Slack is jchen501@asu.edu. Many thanks.